packages/mongoose/src/index.ts

```diff
@@ -0,0 +1,146 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+import { ObjectId } from "mongodb"
+import {
+  userSchema,
+  accountSchema,
+  sessionSchema,
+  verificationTokenSchema,
+} from "./models"
+import type {
+  Adapter,
+  AdapterSession,
+  AdapterUser,
+  VerificationToken,
+} from "next-auth/adapters"
+import type { Account } from "next-auth"
+import dbConnect from "./dbConnect"
+
+export const format = {
+  /** Takes a mongoDB object and returns a plain old JavaScript object */
+  from<T = Record<string, unknown>>(object: Record<string, any>): T {
+    const newObject: Record<string, unknown> = {}
+    for (const key in object) {
```

Comment:
I had got eslint complain about the usage of for..in here. `object.forEach` should be fine right?